opencga-app/app/scripts/azure/arm/mongodb/azuredeploy.json

```diff
@@ -0,0 +1,258 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+        "vmSize": {
+            "type": "string",
+            "metadata": {
+                "description": "Size of vm"
+            },
+            "defaultValue": "Standard_D2_v2"
+        },
+        "dnsLabelPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+        "adminUsername": {
+            "type": "string"
+        },
+        "adminPassword": {
+            "type": "secureString"
+        }
+    },
+    "variables": {
+        "vmName": "mongo-vm",
+        "nsgName": "mongo-nsg",
+        "nicName": "mongo-nic",
+        "publicIPAddressName": "mongo-publicip",
+        "publicIPAddressType": "Dynamic",
```

Comment:
To minimize the risk of failure we should probably move to `Static` here otherwise a redeployment of the ARM might roll the IP Address and cause disconnections for existing clients.

Reply:
(I only know this as I did exactly the same in the VMSS template and the IP kept changing)

Reply:
Changed to Static in https://github.com/opencb/opencga/pull/975/commits/1ec41d9d76237c8182b43e1b401c073730cfff12